internal/conf/configuration.go

```diff
@@ -196,6 +196,7 @@ type ProviderConfiguration struct {
 	Notion                  OAuthProviderConfiguration `json:"notion"`
 	Keycloak                OAuthProviderConfiguration `json:"keycloak"`
 	Linkedin                OAuthProviderConfiguration `json:"linkedin"`
+	LinkedinOIDC            OAuthProviderConfiguration `json:"linkedin_oidc"`
```

Comment:
this makes it such that the env var set is `GOTRUE_EXTERNAL_LINKEDIN_OIDC_ENABLED` instead of `GOTRUE_EXTERNAL_LINKEDINOIDC_ENABLED` 
```suggestion
	LinkedinOIDC            OAuthProviderConfiguration `json:"linkedin_oidc" envconfig:"LINKEDIN_OIDC"`
```